paper/joss/reconst_ivim/paper.bib

```diff
@@ -0,0 +1,153 @@
+@article{le1986mr,
+  title={MR imaging of intravoxel incoherent motions: application to diffusion and perfusion in neurologic disorders.},
+  author={Le Bihan, Denis and Breton, Eric and Lallemand, Denis and Grenier, Philippe and Cabanis, Emmanuel and Laval-Jeantet, Maurice},
+  journal={Radiology},
+  volume={161},
+  number={2},
+  pages={401--407},
+  year={1986}
+}
+
+
+
+@article{LeBihan1986,
+  doi = {10.1148/radiology.161.2.3763909},
+  url = {http://dx.doi.org/10.1148/radiology.161.2.3763909},
+  year  = {1986},
+  month = {nov},
+  publisher = {Radiological Society of North America ({RSNA})},
+  volume = {161},
+  number = {2},
+  pages = {401--407},
+  author = {D Le Bihan and E Breton and D Lallemand and P Grenier and E Cabanis and M Laval-Jeantet},
+  title = {{MR} imaging of intravoxel incoherent motions: application to diffusion and perfusion in neurologic disorders.},
+  journal = {Radiology}
+}
+
+
```

Comment:
Add following article with lable Le1988ra

Le Bihan, D., Breton, E., Lallemand, D., Aubin, M.L., Vignaud, J., Laval-Jeantet, M. (1988). Separation of Diffusion and Perfusion in Intravoxel Incoherent Motion MR Imaging. Radiology 168(2), 497-505. doi: 10.1148/radiology.168.2.3393671